src/components/MTableGroupbar/index.js

```diff
@@ -32,6 +32,26 @@ function MTableGroupbar(props) {
     borderStyle: 'dashed'
   });
 
+  useEffect(() => {
+    if (props.persistentGroupingsId) {
+      const persistentGroupings = [];
+
+      props.groupColumns.forEach((column) => {
```

Comment:
Can we exhacnge that for a map persistentGroupings = props.groupColumns.map(....)? To make the intentions clearer

Reply:
sure, done 👍